Week 2/Weekly Practice Problems (Week 2)/2520: Count the Digits That Divide a Number/elijah.cpp

```diff
@@ -0,0 +1,25 @@
+class Solution {
+public:
+    int countDigits(int num)
+    {
+        int power = 0;
+        int count = 0;
+
+        while (true)
+        {
+            int digit = (int)(num / pow(10, power++)) % 10;
```

Comment:
Ideally, we don't use pow function. 

As you might have noticed, pow returns a floating point number.

We should avoid using with floating point numbers when possible,  to avoid any precision-related errors.

Reply:
Good thinking, I'll keep that in mind for next time. It is interesting that pow() doesn't have an int overload, but oh well.